daemon/cmd/local_node_sync.go

```diff
@@ -25,22 +25,23 @@ import (
 	"github.com/cilium/cilium/pkg/source"
 )
 
-type localNodeInitializerParams struct {
+type localNodeSynchronizerParams struct {
 	cell.In
 
 	Config    *option.DaemonConfig
-	Clientset client.Clientset
 	LocalNode agentK8s.LocalNodeResource
```

Comment:
Reading through this, I think my choice of naming this `LocalNode` is confusing in this context. Perhaps we could rename it to `K8sLocalNode` or `K8sLocalNodeResource`?

Reply:
Renamed to K8sLocalNode.